content/applications/inventory_and_mrp/inventory/product_management/product_replenishment/uom.rst

```diff
@@ -60,55 +58,68 @@ Specify a product's units of measure
 ====================================
 
 To set units of measure on a product, first go to :menuselection:`Inventory --> Products -->
-Products` and click on a product to open its settings. Then, click on :guilabel:`Edit`.
+Products` and select a product to open its settings.
 
 In the :guilabel:`General Information` tab, edit the :guilabel:`Unit of Measure` field to specify
 the unit of measure that the product is sold in. The specified unit will also be the unit used to
 keep track of the product's inventory and internal transfers. Edit the :guilabel:`Purchase UoM`
 field to specify the unit of measure that the product is purchased in.
 
+.. _inventory/product_replenishment/unit-conversion:
+
 Unit conversion
 ===============
 
```

Comment:
Is there a single sentence we can add here as a buffer between the header and the first section? It's OK if not

Reply:
Added a list/summary of the following sections (: